### PR TITLE
Update balloon_time default value

### PR DESCRIPTION
**Pull request by suqinhuang**
100s is a little short for some host, update the default value to 300,
otherwise need to update lots cfg file

Signed-off-by: Suqin Huang <shuang@redhat.com>

id: 1446042

### CODE REVIEW

File: qemu/tests/balloon_boot_in_pause.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def balloon_memory(self, new_mem):
                 raise exceptions.TestFail("Balloon memory fail with error"
                                           " message: %s" % e)
         compare_mem = new_mem
-        balloon_timeout = float(self.params.get("balloon_timeout", 100))
+        balloon_timeout = float(self.params.get("balloon_timeout", 600))
```
Comment (kimi1978wy) [inline qemu/tests/balloon_boot_in_pause.py]: it is 240 enough for "balloon_timeout", pls change it. Thanks

**Comment by kimi1978wy** [comment]
the sleep time has been removed and it can balloon immediately after former evict/enlarge complete  after applying the PR. 

 Acked-by: Yu Wang wyu@redhat.com

**Comment by CongLi** [comment]
Ack.

**Comment by CongLi** [comment]
Thanks all.

**Pull request merged**